.github/scripts/test-info.mjs

```diff
@@ -82,6 +82,20 @@ const components = {
             'AzureBlobStorageQueue',
         ],
     },
+    'bindings.aws.s3': {
```

Comment:
Let's remove this entry as discussed? We don't want to run the tests with docker in conformance and certification test.

Reply:
@berndverst absolutely! I'll take care of that today.